### ISSUE TITLE
inline block

### ISSUE THREAD

**Issue by UXDart** [facebook/draft-js#2366]
**Do you want to request a *feature* or report a *bug*?**
not sure... possible a feature

**What is the current behavior?**
when inserting a custom block, it always uses a new line, I want to use the inserted block in the middle of another blocks

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. You can use this jsfiddle to get started: https://jsfiddle.net/gmertk/e61z7nfa/.**
there are many people asking for this, but there is no specific answer, it seems is possible using custom blocks but not sure how

**What is the expected behavior?**
the inserted block will be between for ex the selected anchor, so there will be text before and after. it can be inserting text or an image for example

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
last version from npm, OSX... not sure if is a bug. a feature or if it ever worked

Thanks

**Comment by max-cross** [comment]
It would be a nice feature. I'm also faced with it and would be great to make atomic blocks inline.

**Comment by krvikash35** [comment]
Yeah..I am also on same boat..can't insert image(atomic block) in between block without splitting that block...

Alteast if some workaround is provided then that will also be very helpful.

**Comment by misakisaysyes** [comment]
here is the inline block solution [issue852](https://github.com/facebook/draft-js/issues/852)
